test/MinterGateway.t.sol

```diff
@@ -1732,6 +1732,8 @@ contract MinterGatewayTests is TestUtils {
 
         uint240 activeOwedM_ = _minterGateway.activeOwedMOf(_minter1);
 
+        if (activeOwedM_ == 0) return; // No penalty if there is no active owed M
```

Comment:
Would be more appropriate to use `vm.assume` for fuzz tests.
`vm.assume(activeOwedM_ != 0); // No penalty if there is no active owed M`

Reply:
added